en/docs/get-started/integration-quick-start-guide.md

```diff
@@ -145,143 +153,207 @@ Go to the `<mi-qsg-home>` directory. The following project files and executable
+
+3.  Update the API name, context, and version if required, and click **Create API**. 
+
+    The overview page of the API that you just created appears. 
+
+    <a href="{{base_path}}/assets/img/integrate/quick-start-guide/new-healthcareapi-view.png"><img src="{{base_path}}/assets/img/integrate/quick-start-guide/new-healthcareapi-view.png" alt="apis list"></a>
+
+4.  Navigate to **Develop -> API Configurations -> Endpoints** from the left menu. You will see that **HTTP/REST Endpoint** is already selected and the production endpoint is already provided .
+
+     Select the `Sandbox Endpoint`, add the endpoint `http://localhost:8290/healthcare`, and **Save**.
+     
+5.  Update the portal configurations and API configurations as required.
+
+     Now, you have successfully created an API using the service.
+
+### Step 3 - Publish the managed API
+
+1. Navigate  to **Deployments** and click **Deploy** to create a revision to deploy in the default Gateway environment. 
+
+2. Navigate to **Lifecycle** and click **Publish** to publish the API in the Gateway environment.
+
+    <a href="{{base_path}}/assets/img/integrate/quick-start-guide/healthcareapi-lifecycle.png"><img src="{{base_path}}/assets/img/integrate/quick-start-guide/healthcareapi-lifecycle.png"></a>
+    
+    If the API is published successfully, the lifecycle state will shift to **PUBLISHED**. 
+
+### Step 4 - Invoke the Managed `HealthcareAPI` via Developer Portal
+
+1. Navigate to  the **Developer Portal** by clicking on the `View In Dev Portal` at the top menu.
+
+    <a href="{{base_path}}/assets/img/integrate/quick-start-guide/view-in-dev-portal.png"><img src="{{base_path}}/assets/img/integrate/quick-start-guide/view-in-dev-portal.png"></a>
+
+2. Sign in using the default username/password `admin/admin`. You will be redirected to the **APIs**.
+
+3. Under **APIs**, you will see the published `HealthcareAPI`. Click on it to navigate to the Overview of the API.
+
+4. Click `Try Out`. This will create a subscription to the API using `Default Application`.
+
+    <a href="{{base_path}}/assets/img/integrate/quick-start-guide/devportal-tryout.png"><img src="{{base_path}}/assets/img/integrate/quick-start-guide/devportal-tryout.png" size="800"></a>
+
+5. Click `GET TEST KEY` to get a test token to invoke the API.
+
+    <a href="{{base_path}}/assets/img/integrate/quick-start-guide/devportal-get-test-key.png"><img src="{{base_path}}/assets/img/integrate/quick-start-guide/devportal-get-test-key.png" size="800"></a>
 
-2. Click the **Hamburger Icon** in the upper-left of the **Publisher**.
-3. Select the **Services** from the menu.
-4. See that the `HealthcareAPI` is listed as a service.
+6. Click **GET** resource `/doctor​/{doctorType}`. Use  **Ophthalmologist** as the doctorType. And Execute.
```

Comment:
```suggestion
6. Click **GET** resource `/doctor​/{doctorType}`. Use  **Ophthalmologist** as the doctorType and click **Execute**.
```

Reply:
Is "Execute" clickable? If so please commit this.

Reply:
fixed.